### PR TITLE
Make sub-accounts stars also migrate and work (i hope)

### PR DESCRIPTION
**Pull request by juan-cortes**
Same as https://github.com/LedgerHQ/lld-v2/pull/230 but working for subaccounts. We still need to change the UI now that we don't have the dnd functionality on hover or at least the cursor to convey that the sidebar items are interactive.

### Type

Bug fix

### CODE REVIEW

File: src/renderer/actions/accounts.js [hunk 1]
```diff
@@ -47,12 +47,22 @@ export const updateAccount: UpdateAccount = payload => ({
...
+export const toggleStarAction: UpdateAccount = (id, parentId) => {
+  return {
+    type: "DB:UPDATE_ACCOUNT",
+    payload: {
+      updater: (account: Account) => {
+        if (parentId && account.subAccounts) {
+          const subAccounts = account.subAccounts.map(sa =>
+            sa.id === id ? { ...sa, starred: !sa.starred } : sa,
+          );
+          return { ...account, subAccounts };
```
Comment (gre) [inline src/renderer/actions/accounts.js:59]: this makes it impossible to star the parent account

**Review by gre — CHANGES_REQUESTED** [review]

File: src/renderer/actions/accounts.js (hunk 1 above)
Comment (gre) [reply src/renderer/actions/accounts.js:59]: nevermind

**Review by gre — APPROVED** [review]
lgtm

**Review by nabil-brn — APPROVED** [review]

**Comment by nabil-brn** [comment]
Before:
Capture d’écran 2020-02-20 à 14 44 21

After:
Capture d’écran 2020-02-20 à 15 21 51


✓ QA-ed

---
e7be249 (HEAD -> starsForSubAccounts, ledger/starsForSubAccounts) Make subaccounts stars also migrate and work (i hope)

**Pull request merged**